app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { withTheme } from 'styled-components'
+import { th } from '@pubsweet/ui'
+import Select from 'react-select'
+
+const Root = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: ${th('gridUnit')};
+`
+
+const SelectLimitMessage = styled.p`
+  color: ${th('colorSuccess')};
+`
+
+const eLifeOptions = [
+  {
+    value: 'biochemistry-chemical-biology',
+    label: 'Biochemistry and Chemical Biology',
+  },
+  { value: 'cancer-biology', label: 'Cancer Biology' },
+  { value: 'cell-biology', label: 'Cell Biology' },
+  {
+    value: 'chromosomes-gene-expression',
+    label: 'Chromosomes and Gene Expression',
+  },
+  {
+    value: 'computational-systems-biology',
+    label: 'Computational and Systems Biology',
+  },
+  {
+    value: 'developmental-biology-stem-cells',
+    label: 'Developmental Biology and Stem Cells',
+  },
+  { value: 'ecology', label: 'Ecology' },
+  {
+    value: 'epidemiology-global-health',
+    label: 'Epidemiology and Global Health',
+  },
+  { value: 'evolutionary-biology', label: 'Evolutionary Biology' },
+  { value: 'genetics-genomics', label: 'Genetics and Genomics' },
+  {
+    value: 'human-biology-medicine',
+    label: 'Human Biology and Medicine',
+  },
+  {
+    value: 'immunology-inflammation',
+    label: 'Immunology and Inflammation',
+  },
+  {
+    value: 'microbiology-infectious-disease',
+    label: 'Microbiology and Infectious Disease',
+  },
+  { value: 'neuroscience', label: 'Neuroscience' },
+  { value: 'physics-living-systems', label: 'Physics of Living Systems' },
+  { value: 'plant-biology', label: 'Plant Biology' },
+  {
+    value: 'structural-biology-molecular-biophysics',
+    label: 'Structural Biology and Molecular Biophysics',
+  },
+]
+
+const findOptionByValue = (value, options) =>
+  options.find(option => option.value === value)
+
+export class ThemelessSubjectAreaDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    const savedOptions = this.props.savedValues.map(value =>
+      findOptionByValue(value, eLifeOptions),
+    )
+
+    this.state = {
+      selectedOptions: savedOptions,
+      hasReachedMultiselectLimit: false,
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleBlur = this.handleBlur.bind(this)
+
+    this.customReactSelectStyles = {
+      multiValue: (base, state) => ({
+        ...base,
+        backgroundColor: this.props.theme.colorPrimary,
+      }),
+    }
+  }
+
+  // https://github.com/JedWatson/react-select/blob/v2/src/Select.js
+  // react-select exposes onChange: (ValueType, ActionMeta) => void
+  handleChange = selectedOptions => {
+    if (selectedOptions.length <= 2) {
+      this.setState({ selectedOptions, hasReachedMultiselectLimit: false })
+      if (this.props.onChange) this.props.onChange(selectedOptions)
```

Comment:
`if` unnecessary

Reply:
Ah yeah, this was when I was imagining creating a MultiSelectDropdown component that was more generic, rather than SubjectAreaDropdown, which is very specific to this context. So my thinking was that `if (this.props.onChange)` would be good for the same reasons that you and I put it into pubsweet's Menu component - https://gitlab.coko.foundation/pubsweet/pubsweet/merge_requests/201/diffs#83853fc5eb0bcac69c38969965b30fa045468482_165_163

But you're right, since SubjectAreaDropdown is totally context specific & `onChange` is required, this is meaningless :+1: